definitions/npm/react-intl_v2.x.x/flow_v0.57.x-/test_react-intl_v2.x.x.js

```diff
@@ -0,0 +1,206 @@
+/* @flow */
+import * as React from "react";
+import {
+  intlShape,
+  addLocaleData,
+  defineMessages,
+  injectIntl,
+  FormattedMessage,
+  FormattedHTMLMessage,
+  FormattedDate,
+  FormattedTime,
+  FormattedRelative,
+  FormattedNumber,
+  FormattedPlural,
+  IntlProvider
+} from "react-intl";
+import type { IntlShape, MessageDescriptor, IntlInjectedComponentClass } from "react-intl";
```

Comment:
Import `IntlInjectedComponentClass` for testing.
Original test included exporting an `injectIntl`-ed component without specifying the type explicitly and trying to render it without passing `defaultProps`.